examples/algebraic_constraints.py

```diff
@@ -0,0 +1,44 @@
+import desbordante
+import pandas
+
+TABLE = 'datasets/cargo_march.csv'
+HEADER = 0
+SEPARATOR = ','
+P_FUZZ = 0.85
+FUZZINESS = 0.2
+BUMPS_LIMIT = 0
+WEIGHT = 0.1
+BIN_OPERATION = '-'
+AC_SEED = 11
+ITERATIONS_LIMIT = 4
+
+algo = desbordante.ACAlgorithm()
+
+df = pandas.read_csv(TABLE, sep=SEPARATOR, header=HEADER)
+df_without_id = df[['Delivery date', 'Dispatch date']]
```

Comment:
```suggestion
df = pandas.read_csv(TABLE, sep=SEPARATOR, header=HEADER, index_col=['id'])
```

Reply:
I don't think I want it like that. Those are id's of cargos, it just happens so that they are identical to row indices

Reply:
```suggestion
df = pandas.read_csv(TABLE, sep=SEPARATOR, header=HEADER, use_cols=['Delivery date', 'Dispatch date'])
```

Reply:
Very nice

Reply:
But I later use values from id column

Reply:
Then I guess that's fine